### ISSUE TITLE
lxqt-config-monitor: monitor setup not restored

### ISSUE THREAD

**Issue by HeikoTietze** [lxde/lxqt#1096]
My setup with
* eDP1 connected 1368x768+1920+300 (normal left inverted right x axis y axis) 310mm x 170mm 
* HDMI1 connected 1920x1080+0+0 (normal left inverted right x axis y axis) 600mm x 340mm
is never restored on restart. 

It starts always with 
* eDP1 connected 1600x900+0+0 (normal left inverted right x axis y axis) 310mm x 170mm
* HDMI1 connected 1440x900+0+0 (normal left inverted right x axis y axis) 600mm x 340mm
(output from xrandr)

Window manager is kwin_x11, running on an Intel HD4600 graphic card. Same (or similar) issue brought me from KDE to LXQt (all plasma tools are still on my system, including kscreen). What can I do?

**Comment by pmattern** [comment]
Confirmed.
Running latest VCS checkouts on up to date Arch Linux restoring screen settings doesn't work any longer. `lxqt-config-monitor` still behaves as usual, in particular `Save` still creates file `~/.config/autostart/lxqt-config-monitor-autostart.desktop` which doesn't have an effect any longer, though.

The problem affects single monitor systems and Openbox as well.
It was introduced by upgrading libscreen from 5.6.5 to 5.7.0. The only possibly relevant message in the logs (systemd journal) is
```
kscreen: canBeApplied: The configuration is too wide: 0
```
which isn't exactly plausible as the very same resolution works flawlessly if set by `lxqt-config-monitor`.
Cannot tell whether it's a bug in KScreen or we just need to adopt to the new version.

Basically this should be addressed before releasing 0.11, feasible of course only if the problem is not in KScreen as said above.

**Comment by HeikoTietze** [comment]
Happy to get a confirmation, at least ;-)
Cross-linking with KDE bug https://bugs.kde.org/show_bug.cgi?id=365346

**Comment by palinek** [comment]
Have a look on lxde/lxqt-config#95
